DeeployTest/Platforms/Snitch/main.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * ----------------------------------------------------------------------
+ *
+ * File: deeploytest.c
+ *
+ * Last edited: 23.04.2024
+ *
+ * Copyright (C) 2024, ETH Zurich and University of Bologna.
+ *
+ * Author: Philip Wiese (wiesep@iis.ee.ethz.ch), ETH Zurich
+ *
+ * ----------------------------------------------------------------------
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "CycleCounter.h"
+#include "Network.h"
+#include "snrt.h"
+#include "testinputs.h"
+#include "testoutputs.h"
+
+int main(void) {
+
+  uint32_t core_id = snrt_global_core_idx();
+  uint32_t compute_core_id = snrt_global_compute_core_idx();
+
+#ifdef BANSHEE_SIMULATION
```

Comment:
Change this macro name to smth more general like SIMULATORS, since we are using GVSoC too.

Reply:
you can check it here https://github.com/pulp-platform/Deeploy/pull/14/commits/fe85e5c5f6492e2fdf106424b0ac3c7b800e902c